docs/proposals/20250103-qwen2.5-coder.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Proposal for new model support
+authors:
+  - "Qinghui Zhuang"
+reviewers:
+  - "Kaito contributor"
+creation-date: 2025-01-03
+last-updated: 2025-01-03
+status: provisional
+---
+
+# Title
+Add Qwen 2.5 Coder to Kaito supported model list
+
+## Glossary
+N/A
+
+## Summary
+- **Model description**: Qwen2.5-Coder is the latest series of Code-Specific Qwen large language models (formerly known as CodeQwen). It brings significantly improvements in code generation, code reasoning and code fixing with long-context support up to 128K tokens. For more information, refer to the [Qwen2.5 Documentation](https://qwenlm.github.io/blog/qwen2.5/) and access the model on [Hugging Face](https://huggingface.co/Qwen/Qwen2.5-Coder-7B-Instruct).
+- **Model usage statistics**: In the past month, Mistral-7B-Instruct-v0.1 has garnered 118,568 downloads on Hugging Face, reflecting its widespread popularity. Google Trends data shows a high level of search interest in <!-- markdown-link-check-disable --> ["qwen2.5"](https://trends.google.com/trends/explore?q=qwen2.5), <!-- markdown-link-check-enable --> indicating strong market curiosity. 
```

Comment:
"Mistral-7B-Instruct-v0.1" , is this wrong?

Reply:
sorry